spec/support/api_spec_helper.rb

```diff
@@ -6,47 +6,32 @@
 require 'json'
 
 module ApiSpecHelper
-  HEADER_ALIASES = {
-    "auth_token" => "HTTP_X_AUTH_TOKEN",
-    "miq_token"  => "HTTP_X_MIQ_TOKEN",
-    "miq_group"  => "HTTP_X_MIQ_GROUP"
-  }
-
-  DEF_HEADERS = {
-    "Content-Type" => "application/json",
-    "Accept"       => "application/json"
-  }
-
-  def update_headers(headers)
-    HEADER_ALIASES.keys.each do |k|
-      if headers.key?(k)
-        headers[HEADER_ALIASES[k]] = headers[k]
-        headers.delete(k)
-      end
-    end
-    headers.merge!("HTTP_AUTHORIZATION" => @http_authorization) if @http_authorization
-    headers.merge(DEF_HEADERS)
+  def request_headers
+    @request_headers ||= {
+      "Content-Type" => "application/json",
+      "Accept"       => "application/json"
+    }
```

Comment:
Not sure why you moved this away form a constant...this will actually recreate the same hash on every request.  Might be nicer to have a frozen constant and then the method can just return the frozen constant.